build/ci/phase-template.yml

```diff
@@ -33,13 +33,13 @@ phases:
           ${{ insert }}: ${{ parameters.customMatrixes }}
       ${{ insert }}: ${{ parameters.queue }}
     steps:
+    - ${{ if eq(parameters.name, 'MacOS') }}:
```

Comment:
Note - you are going to need to update the "official" build as well:

https://github.com/dotnet/machinelearning/blob/6f4e055b113b2bac024b7241ba469b9c20453fbe/build/vsts-ci.yml#L37-L60

Reply:
thanks - ill update that as well.